src/main/java/no/digipost/time/JavaClockAccessors.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (C) Posten Norge AS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package no.digipost.time;
+
+import java.time.Instant;
+import java.time.ZoneId;
+
+/**
+ * This is simply a duplication of the <em>time accessor</em> API
+ * of {@link java.time.Clock}.
+ */
+interface JavaClockAccessors {
```

Comment:
Kan kanskje JavaClockAccessor og JavaClockAdditionalAccessors bli slått sammen?

Reply:
De er separate interfaces for å ikke eksponere mer API til ting enn hva som strengt tatt trengs.
`JavaClockAdjuster` arver fra `JavaClockAccessors`, den trenger ikke tilleggs-accessorene som er i `JavaClockAdditionalAccessors`

Reply:
Ah right, I didn't see the that `JavaClockAdjuster` used it.